common/common.go

```diff
@@ -70,7 +70,7 @@ func GetLogrusCaller() *stack.FrameInfo {
 	var frames [32]uintptr
 
 	// iterate until we find non logrus function
-	length := runtime.Callers(3, frames[:])
+	length := runtime.Callers(5, frames[:])
```

Comment:
There is a previous commit that changed this from 5 to 3 along with some other changes. I don't recall why I changed this, but the tests all pass so I have no objections. 

Thanks!